src/components/AssetSelector.tsx

```diff
@@ -98,6 +98,7 @@ function AssetSelector({
         disabled={disabled}
         searchable={searchable}
         onChange={(option: TAssetOption) => onSelect(option)}
+        getOptionLabel={(option) => (showOnlySymbol ? option.ticker : option.name)}
```

Comment:
Can we add a test for this case plz?
https://testing-library.com/docs/example-input-event